controls/raddocking/how-to/add-menu-items-to-the-radpanes-menu.md

```diff
@@ -14,136 +14,220 @@ The purpose of this tutorial is to show you how to customize the __RadPane__ men
-		
-		        <telerik:RadMenuItem 
-		            IsChecked="{Binding IsDockableOptionChecked}" 
-		            Command="telerik:RadDockingCommands.Dockable" 
-		            CommandParameter="{Binding}"
-		            CommandTarget="{Binding}"
-		            Header="{Binding Command.Text, RelativeSource={RelativeSource Self}}" />
-		
-		        <telerik:RadMenuItem 
-		            Command="telerik:RadDockingCommands.CloseAllPanesButThisCommand"
-		CommandParameter="{Binding}" 
-		            CommandTarget="{Binding}"
-		            Header="{Binding Command.Text, RelativeSource={RelativeSource Self}}" />
-		    </telerik:RadContextMenu>
-		</DataTemplate>
-		
-		<Style TargetType="telerik:RadPane">
-		    <Setter Property="ContextMenuTemplate" Value="{StaticResource ContextMenuTemplate}" />
-		</Style>
+		<Window.Resources>
+			<DataTemplate x:Key="ContextMenuTemplate">
+				<telerik:RadContextMenu InheritDataContext="False">
+					<telerik:RadMenuItem 
+						IsChecked="{Binding IsFloatingOnly}"
+						Command="telerik:RadDockingCommands.Floating" 
+						CommandParameter="{Binding}"                
+						CommandTarget="{Binding}"
+						Header="{Binding Command.Text, RelativeSource={RelativeSource Self}}" />
+			
+					<telerik:RadMenuItem 
+						IsChecked="{Binding IsDockableOptionChecked}" 
+						Command="telerik:RadDockingCommands.Dockable" 
+						CommandParameter="{Binding}"
+						CommandTarget="{Binding}"
+						Header="{Binding Command.Text, RelativeSource={RelativeSource Self}}" />
+			
+					<telerik:RadMenuItem 
+						Command="telerik:RadDockingCommands.CloseAllPanesButThisCommand"
+			CommandParameter="{Binding}" 
+						CommandTarget="{Binding}"
+						Header="{Binding Command.Text, RelativeSource={RelativeSource Self}}" />
+				</telerik:RadContextMenu>
+			</DataTemplate>
+			
+			<Style TargetType="telerik:RadPane">
+				<Setter Property="ContextMenuTemplate" Value="{StaticResource ContextMenuTemplate}" />
+			</Style>
+		</Window.Resources>
+		<Grid>
+			<telerik:RadDocking x:Name="radDocking">
+				<telerik:RadSplitContainer>
+					<telerik:RadPaneGroup>
+						<telerik:RadPane Title="Pane 1" Content="Pane 1 Content" />
+						<telerik:RadPane Title="Pane 2" Content="Pane 2 Content" />
+						<telerik:RadPane Title="Pane 3" Content="Pane 3 Content" />
+						<telerik:RadPane Title="Pane 4" Content="Pane 4 Content" />
+					</telerik:RadPaneGroup>
+				</telerik:RadSplitContainer>
+			</telerik:RadDocking>
+		</Grid>
 	{{endregion}}
 
-1. You should implement your custom logic in the __OnCloseAllPanesButThis__ and __OnCloseAllPanesButThisCanExecute__ event handlers. The following code-snippet representis a sample logic for the handlers. 
+1. You should implement your custom logic in the __OnCloseAllPanesButThis__ and __OnCloseAllPanesButThisCanExecute__ command Execute and CanExecute methods. __Example 3__ demonstrate a sample logic for the command methods. 

```

Comment:
We can rephrase the first sentence a little. For example:
You should implement your custom logic in the __OnCloseAllPanesButThis__ and __OnCloseAllPanesButThisCanExecute methods. They specify whether the command can be executed and what action is performed, when it is executed.

Also the 2nd sentense does not need "a":
..demonstrate sample logic ...